sheetfu/helpers.py

```diff
@@ -31,6 +31,19 @@ def convert_coordinates_to_a1(row, column, number_of_row=1, number_of_column=1,
     return notation
 
 
+def append_sheet_name(a1_string, sheet_name):
+    if '!' in a1_string:
+        a1_sheet_name, cells_a1_string = a1_string.split('!')
```

Comment:
Well, but I wanted to make sure that it's a "valid" A1 notation string, with the sheet_name in a1_string it would still be valid even if I pass something like:

people_random_string_huehue

(with people being the sheet)

Reply:
Ok. That's a good reason! 👍